sdk/ai/Azure.AI.Inference/samples/Sample3_ChatCompletionsStreaming.md

```diff
@@ -0,0 +1,63 @@
+# Chat Completions with Streaming Response
+
+This sample demonstrates how to get a chat completion streaming response from the service using a synchronous call.
+
+## Usage
+
+Set these two environment variables before running the sample:
+
+1. AZURE_AI_CHAT_ENDPOINT - Your endpoint URL, in the form `https://your-deployment-name.your-azure-region.inference.ai.azure.com` where `your-deployment-name` is your unique AI Model deployment name, and `your-azure-region` is the Azure region where your model is deployed.
+
+2. AZURE_AI_CHAT_KEY - Your model key (a 32-character string). Keep it secret.
+
+```C# Snippet:Azure_AI_Inference_HelloWorldStreamingScenarioAsync
+var endpoint = new Uri(System.Environment.GetEnvironmentVariable("AZURE_AI_CHAT_ENDPOINT"));
+var credential = new AzureKeyCredential(System.Environment.GetEnvironmentVariable("AZURE_AI_CHAT_KEY"));
+
+var client = new ChatCompletionsClient(endpoint, credential, new ChatCompletionsClientOptions());
+
+var requestOptions = new ChatCompletionsOptions()
+{
+    Messages =
+    {
+        new ChatRequestSystemMessage("You are a helpful assistant."),
+        new ChatRequestUserMessage("How many feet are in a mile?"),
+    },
+};
+
+StreamingResponse<StreamingChatCompletionsUpdate> response = await client.CompleteStreamingAsync(requestOptions);
```

Comment:
Should this have been synchronous call?

Reply:
No, streaming only currently really does async handling, so we don't have a synchronous example